source/common/stats/utility.cc

```diff
@@ -34,5 +35,84 @@ absl::optional<StatName> Utility::findTag(const Metric& metric, StatName find_ta
   return value;
 }
 
+namespace {
+
+// Helper class for the three Utility::*FromElements implementations to build up
+// a joined StatName from a mix of StatName and string_view.
+struct ElementVisitor {
+  ElementVisitor(SymbolTable& symbol_table) : symbol_table_(symbol_table), pool_(symbol_table) {}
```

Comment:
`explicit`

Reply:
I made it a 2-arg constructor per your other suggestion, so no need.